src/vs/workbench/contrib/terminalContrib/chatAgentTools/browser/tools/runInTerminalTool.ts

```diff
@@ -641,6 +643,14 @@ export class RunInTerminalTool extends Disposable implements IToolImpl {
 		return toolTerminal;
 	}
 
+	private _registerInputListener(toolTerminal: IToolTerminal): void {
+		const disposable = toolTerminal.instance.onData(data => {
+			toolTerminal.receivedUserInput = data.length > 0;
+		});
+		this._register(toolTerminal.instance.onDisposed(() => disposable.dispose()));
+	}
```

Comment:
You should filter out `telemetryIgnoredSequences`

https://github.com/microsoft/vscode/blob/e2fb54b517a2180f3710a9853c43e5f9ad0848bc/src/vs/workbench/contrib/terminalContrib/chatAgentTools/browser/tools/runInTerminalTool.ts#L129-L132